arm-insights/2015-05-01/swagger/ApplicationInsights_ComponentClient.json

```diff
@@ -0,0 +1,540 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "ApplicationInsights.Client.Component",
+    "description": "Azure Application Insights client for Components.",
+    "version": "2015-05-01"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow.",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/microsoft.insights/components": {
+      "get": {
+        "description": "Gets a list of all Application Insights components within a subscription.",
+        "operationId": "Component_List",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "A list containing 0 or more Application Insights component definitions.",
+            "schema": {
+              "$ref": "#/definitions/ApplicationInsightsComponentListResult"
+            },
+            "examples": {
```

Comment:
instead of providing example in the spec, can you please provide them using the "x-ms-examples" extension. 
Take a look at the RedisCache swagger spec as an example over [here](https://github.com/Azure/azure-rest-api-specs/blob/master/arm-redis/2016-04-01/swagger/redis.json#L43).